eras/alonzo/test-suite/cardano-ledger-alonzo-test.cabal

```
@@ -1,6 +1,6 @@
 cabal-version: 3.0
 name:          cardano-ledger-alonzo-test
-version:       1.2.1.4
+version:       1.2.2.0
```

There is no need to bump the version:
* patch version was previously bumped
* minor version only needs to be bumped whenever there are user facing additions to the API, which was not the case in #4607
```suggestion
version:       1.2.1.4
```